funnel/forms/session.py

```diff
@@ -21,3 +23,11 @@ class SessionForm(forms.Form):
     featured = forms.BooleanField(__("This is a featured session"), default=False)
     start_at = forms.HiddenField(__("Start Time"), validators=[forms.validators.DataRequired()])
     end_at = forms.HiddenField(__("End Time"), validators=[forms.validators.DataRequired()])
+
+
+class SavedSessionForm(forms.Form):
+    save = forms.BooleanField(
+        __("Save this session?"),
+        validators=[forms.validators.InputRequired()],
```

Comment:
This validator cannot be used for HTML form submissions. It's JSON-only. Your options are:

1. Don't support HTML forms at all, requiring JSON-only (and thereby JS-only)
2. Drop the validator so that it works for both. However, the absence of a `save` attribute will imply removal, which will be unexpected behaviour for a JSON submission.
3. Use a `SelectField` with choices False and True, and use `coerce=coaster.utils.getbool` so that it works for HTML form submission.